src/components/mods/modsTable.tsx

```diff
@@ -370,28 +458,52 @@ export const ModsTable = ({ qualities, difficulties, modsWithInfo, isLoading }:
 
 
             if (
-                mapCountRange[0] !== undefined ||
-                mapCountRange[1] !== undefined
+                selectedModTypes.length &&
+                !selectedModTypes.includes(modWithInfo.type)
+            ) {
+                return false;
+            }
+
+
+            if (
+                debouncedPublisherQuery &&
+                !modWithInfo.publisherName.toLowerCase().includes(debouncedPublisherQuery.trim().toLowerCase())
+            ) {
+                return false;
+            }
+
+
+            if (
```

Comment:
This `if` isn't required, you anyway check for undefined inside. You can remove this `if` and bring out the inner checks.

Reply:
Good catch. I'll fix.